apps.yaml

```
@@ -125,3 +125,13 @@ clease-gui:
     - data-analysis
     - utilities
     - technology-ase
+rs@md:
```

Unfortunately app names must match the following regular expression: `[A-Za-z][A-Za-z0-9-_]`.
Maybe, this would be acceptable?
```suggestion
rs-at-md:
```
The title should be fine with the `@` included.

---

Thanks for the note. I have now changed the name.